src/lavinmq/queue/stream_queue_message_store.cr

```diff
@@ -88,6 +106,71 @@ module LavinMQ
         {msg_offset, segment, pos}
       end
 
+      def last_offset_by_consumer_tag(consumer_tag : String)
+        if pos = @consumer_offset_positions[consumer_tag]?
+          tx = @consumer_offsets.to_slice(pos, 8)
+          return IO::ByteFormat::SystemEndian.decode(Int64, tx)
```

Comment:
Not that LavinMQ will work on any BigEndian platform that I know of but still, we probably want to set it to a fixed endian so that the data_dir can be moved between machines (also for replication)
 
```suggestion
          return IO::ByteFormat::LittleEndian.decode(Int64, tx)
```